org.osgi.test.common/src/main/java/org/osgi/test/common/bitmaps/Bitmap.java

```diff
@@ -126,4 +126,8 @@ public String maskToString(int mask) {
 		}
 		return mask + ":" + bits.collect(Collectors.joining(" | "));
 	}
+
+	public static boolean typeMatchesMask(int type, int mask) {
```

Comment:
Shouldn't you bump the package version to 1.1?

Reply:
sure. i pushed to the wrong repo.